docs/src/pages/components/masonry/BasicMasonry.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+
+export default function BasicMasonry(): JSX.Element {
+  return (
+    <Masonry cols={4} spacing={1}>
+      {heights.map((height, idx) => (
```

Comment:
A convention we use in the codebase
```suggestion
      {heights.map((height, index) => (
```

x the other cases